dataflow/templates/README.md

```diff
@@ -0,0 +1,90 @@
+# Cloud Dataflow Templates
+
+Samples showing how to create and run an [Apache Beam][] on [Google Cloud Dataflow][].
+
+## Setup
+
+Setup instructions assume you have an active Google Cloud Project and with an associated billing account.
+The following instructions will help you prepare your development environment.
+
+1. Install [Cloud SDK][].
+1. Setup the Cloud SDK
+
+   ```bash
+   gcloud init
```

Comment:
Well, that's nice, might there be more?  Create a project perhaps?  Enable some API's?